website/docs/guides/oidc.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Common OIDC provider configurations
```

Comment:
We could reference the guide in the [WGE install OIDC section](https://github.com/weaveworks/weave-gitops/blob/d8deb479d394816f067e6e68fdfbd0fd02b0f4da/website/docs/enterprise/getting-started/install-enterprise.mdx#L126) so the user installing could already benefit from it.